recipe/meta.yaml

```diff
@@ -11,35 +11,67 @@ source:
 
 build:
   noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
 
+# This *really* shouldn't be necessary
```

Comment:
I really *don't* understand why this block was necessary; it's not it the matplotlib or raider feedstocks, but is in nbconvert??

Anyways, it's working so whatevs.